src/vue-dni.spec.js

```diff
@@ -1,4 +1,4 @@
-import Vue from 'vue';
+import Vue from 'vue/dist/vue.common';
```

Comment:
Unable to resolve path to module 'vue/dist/vue.common'  import/no-unresolved

Reply:
Creo que esto es porque está como dev dependency, pero funciona al desarrollar.